apps/Arm/vm_qemu_virtio/src/trace.c

```diff
@@ -82,6 +155,7 @@ void trace_stop(void)
      trace_started = false;
 }
 
+/* #TODO: do we need this? to be removed */
```

Comment:
Yes, we do need this: this dumps pointers to seL4 threads' TCBs and their names. So that way you can (and in continuation task) should decode the thread names so that in kernelshark diagram we can see human readable names for seL4 threads also.

Reply:
Comment removed =)